napari/utils/settings/_defaults.py

```diff
@@ -0,0 +1,89 @@
+"""Settings management.
+"""
+
+from enum import Enum
+from typing import List, Tuple
+
+from pydantic import BaseSettings, Field
+
+from napari.utils.events.evented_model import EventedModel
+
+
+class QtBindingEnum(str, Enum):
+    """Python Qt binding to use with the application."""
+
+    pyside = 'pyside2'
+    pyqt = 'pyqt5'
+
+
+class ThemeEnum(str, Enum):
+    """Application color theme."""
+
+    dark = 'dark'
+    light = 'light'
+
+
+class ApplicationSettings(BaseSettings, EventedModel):
+    """Main application settings."""
+
+    version = (0, 1, 0)
```

Comment:
is this the settings schema version?  would it make sense to name it something like `schema_version` to avoid confusion with `napari` version?  when will/should this version be changed?  anytime an attribute is added?  or just a backwards incompatible one?  Can you add a comment to the code with guidance on that?

Reply:
Fixing